base/Dockerfile

```diff
@@ -7,10 +7,17 @@ RUN apt-get update -y && apt-get install --no-install-recommends -y -q curl pyth
 
 # Install the latest LTS release of nodejs
 RUN mkdir /nodejs && curl https://nodejs.org/dist/v4.2.2/node-v4.2.2-linux-x64.tar.gz | tar xvzf - -C /nodejs --strip-components=1
+ENV PATH $PATH:/nodejs/bin
+
+# Install semver, as required by the node version install script.
+RUN npm install https://storage.googleapis.com/gae_node_packages/semver.tar.gz
+
+# Add the node version install script
+ADD install_node /usr/local/bin/install_node
```

Comment:
It is, and when I tested the image it appears to carry over the flag.

On Thu, Dec 3, 2015, 6:05 PM dlorenc <notifications@github.com> wrote:

> In base/Dockerfile
> <https://github.com/GoogleCloudPlatform/nodejs-docker/pull/40#discussion_r46641407>
> :
>
> > @@ -7,10 +7,17 @@ RUN apt-get update -y && apt-get install --no-install-recommends -y -q curl pyth
> >
> >  # Install the latest LTS release of nodejs
> >  RUN mkdir /nodejs && curl https://nodejs.org/dist/v4.2.2/node-v4.2.2-linux-x64.tar.gz | tar xvzf - -C /nodejs --strip-components=1
> > +ENV PATH $PATH:/nodejs/bin
> > +
> > +# Install semver, as required by the node version install script.
> > +RUN npm install https://storage.googleapis.com/gae_node_packages/semver.tar.gz
> > +
> > +# Add the node version install script
> > +ADD install_node /usr/local/bin/install_node
>
> Just to double check, this file is marked executable in your branch? I
> can't really find a way for github to tell me. If not it might be safer to
> chmod +x it as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/nodejs-docker/pull/40/files#r46641407>
> .
>